RocketBot2/RocketBot2.csproj

```diff
@@ -1082,7 +1082,7 @@ foreach (var item in filesToCleanup)
   </Target>
   <PropertyGroup>
     <PostBuildEvent>cd "$(TargetDir)"
-copy /y /b RocketBot2.exe "$(SolutionDir)\PoGo.Necrobot.Window\$(OutDir)\RocketBot2.exe" /b
+REM copy /y /b RocketBot2.exe "$(SolutionDir)\PoGo.Necrobot.Window\$(OutDir)\RocketBot2.exe" /b
```

Comment:
@CDAGaming maybe this in gui project  file ``copy /y /b "$(SolutionDir)\RocketBot2\$(OutDir)\RocketBot2.exe" /b
"$(TargetDir)\RocketBot2.exe" /b``